src/System.Runtime.WindowsRuntime/src/System/Threading/Tasks/AsyncInfoToTaskBridge.dotnet53.cs

```diff
@@ -198,6 +203,10 @@ private void Complete(IAsyncInfo asyncInfo, Func<IAsyncInfo, TResult> getResults
                     switch (asyncStatus)
                     {
                         case AsyncStatus.Completed:
+                            if (AsyncCausalityTracer.LoggingOn)
+                            {
+                                AsyncCausalityTracer.TraceOperationCompletion(CausalityTraceLevel.Required, base.Task.Id, AsyncCausalityStatus.Completed);
+                            }
```

Comment:
Why do we only trace when the operation completes successfully... we don't need to do so as well when it completes as Error or Canceled?  I see that this does match what's being done in the full framework, but I don't remember (or maybe never knew) why it was done that way.